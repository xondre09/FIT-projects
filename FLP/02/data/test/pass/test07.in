S a B R
S b B R
S   B R
B a S a
B b S b
B   C  
C a D L
C b D L
C   D L
D a C b
D b E a
E a G L
E b G L
E   G L
G a E a
G b E b
G   X R
H a I R
H b I R
H   I R
I a J a
I b L b
I   Z L
J a K L
J b K L
J   K L
K a O a
K b O a
K   O a
L a M L
L b M L
L   M L
M a O b
M b O b
M   O b
O a H R
O b H R
O   H R
X a H  
X b Y b
Y a U L
Y b U L
Y   U L
U a Y a
U b Y b
U   F  
Z a Y  
Z b Y  
Z   Y  
 baaaaa
