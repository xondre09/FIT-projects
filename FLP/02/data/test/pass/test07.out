S baaaaa
 Bbaaaaa
 Sbaaaaa
 bBaaaaa
 bSaaaaa
 baBaaaa
 baSaaaa
 baaBaaa
 baaSaaa
 baaaBaa
 baaaSaa
 baaaaBa
 baaaaSa
 baaaaaB 
 baaaaaC 
 baaaaDa
 baaaaCb
 baaaDab
 baaaCbb
 baaDabb
 baaCbbb
 baDabbb
 baCbbbb
 bDabbbb
 bCbbbbb
 Dbbbbbb
 Eabbbbb
G abbbbb
 Xabbbbb
 H bbbbb
  Ibbbbb
  Lbbbbb
 M bbbbb
 Obbbbbb
 bHbbbbb
 bbIbbbb
 bbLbbbb
 bMbbbbb
 bObbbbb
 bbHbbbb
 bbbIbbb
 bbbLbbb
 bbMbbbb
 bbObbbb
 bbbHbbb
 bbbbIbb
 bbbbLbb
 bbbMbbb
 bbbObbb
 bbbbHbb
 bbbbbIb
 bbbbbLb
 bbbbMbb
 bbbbObb
 bbbbbHb
 bbbbbbI 
 bbbbbZb
 bbbbbY 
 bbbbUb
 bbbbYb
 bbbUbb
 bbbYbb
 bbUbbb
 bbYbbb
 bUbbbb
 bYbbbb
 Ubbbbb
 Ybbbbb
U bbbbb
F bbbbb
